.travis.yml

```diff
@@ -0,0 +1,57 @@
+language: android
+jdk: oraclejdk8
+sudo: true
+env:
+  matrix:
+  - ANDROID_TARGET=android-21 ANDROID_ABI=armeabi-v7a
```

Comment:
These settings are to setup the android virtual device (avd) for testing.  The `android-21` is an avd id.  There's a command in `before_script` section that uses these VARs.  It's basically for testing so actually we'll need these after all.

- echo no | android create avd --force --name test --target $ANDROID_TARGET --abi $ANDROID_ABI

Josh is correct about the matrix keyword.  It's not needed at this point, but I'm inclined to just leave it in so that you can easily (as josh said) run tests against multiple targets.